src/pages/experiments/[experimentId]/plots-and-tables/frequency/components/SelectData.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Form,
+  Radio,
+  Select,
+} from 'antd';
+
+const { Option } = Select;
+const SelectData = (props) => {
+  const { onUpdate, config } = props;
+
+  const changeClusters = (val) => {
+    onUpdate({ chosenClusters: val });
+  };
+  return (
+    <>
+      <Form.Item>
+        <Select value={config.chosenClusters} onChange={(value) => changeClusters(value)}>
+          <Option value='louvain'>Louvain clusters</Option>
+          <Option value='scratchpad'>Scratchpad</Option>
+        </Select>
+      </Form.Item>
+    </>
+  );
+};
```

Comment:
for this, you can essentially reuse the same code as this in embedding-categorical index.js file:

```
<Panel header='Group by' key='6'>
              <p>
                Select the cell set category you would like to group cells by.
              </p>
              <Space direction='vertical' style={{ width: '100%' }}>
                <Select
                  labelInValue
                  style={{ width: '100%' }}
                  placeholder='Select cell set...'
                  value={{ key: config.selectedCellSet }}
                  options={generateCellSetOptions()}
                  onChange={onCellSetSelect}
                />
              </Space>
            </Panel>
```

Can you modify this `SelectData` component to basically contain this code and then also refactor the embedding-categorical index.js file to render `SelectData`?